test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1041,13 +1077,54 @@ public static void forceFail(String reason) {
         latch.countDown();
     }
 
+    /**
+     * Adds a {@code message} to the log area, if enabled by
+     * {@link Builder#logArea()} or {@link Builder#logArea(int)}.
+     *
+     * @param message to log
+     */
+    public static void log(String message) {
+        invokeOnEDTUncheckedException(() -> {
+            if (logArea != null) {
```

Comment:
Should it throw `NullPointerException`? It's a developer error. If logging is used, it should be enabled; therefore throwing an exception rather than hiding the problem is reasonable, isn't it? Otherwise, the problem could remain unnoticed.

Reply:
Good point, updated.